### PR TITLE
Change gene names in expression files to strings

### PR DESCRIPTION
**Pull request by JenkoB**
@jkokosar @dblenkus please review

### CODE REVIEW

File: resolwe_bio/processes/clustering/hierarchical_clustering.yml [hunk 1]
```diff
@@ -66,7 +66,7 @@
   requirements:
     expression-engine: jinja
   data_name: 'Hierarchical clustering of samples'
-  version: 1.0.8
+  version: 1.0.9
```
Comment (jkokosar) [inline resolwe_bio/processes/clustering/hierarchical_clustering.yml]: If only the called script was changed, there is no need to bump the version.

File: resolwe_bio/tests/processes/test_clustering.py [hunk 2]
```diff
@@ -18,7 +18,8 @@ def test_hc_clustering(self):
             'genes': ['DPU_G0067096', 'DPU_G0067098', 'DPU_G0067100']}
         self.run_process('clustering-hierarchical-genes-etc', inputs)
 
-    def test_hc_clustering_samples(self):
+    def test_hc_clustering_samples_ucsc(self):
+        self.keep_all()
```
Comment (jkokosar) [inline resolwe_bio/tests/processes/test_clustering.py]: remove this

File: resolwe_bio/processes/clustering/hierarchical_clustering.yml [hunk 3]
```diff
@@ -134,7 +134,7 @@
   requirements:
     expression-engine: jinja
   data_name: 'Hierarchical clustering of genes'
-  version: 1.0.3
+  version: 1.0.4
```
Comment (jkokosar) [inline resolwe_bio/processes/clustering/hierarchical_clustering.yml]: This one also does not need to be updated.

**Comment by jkokosar** [comment]
Wait for your other PR to be merged, than rebase this one.

**Comment by mstajdohar** [comment]
Rebase to master

**Pull request merged**